packages/token-search-discovery-controller/src/token-discovery-api-service/token-discovery-api-service.ts

```diff
@@ -39,4 +44,60 @@ export class TokenDiscoveryApiService extends AbstractTokenDiscoveryApiService {
 
     return response.json();
   }
+
+  async getTopLosersByChains(
+    topLosersParams: TopLosersParams,
+  ): Promise<MoralisTokenResponseItem[]> {
+    const url = new URL('/tokens-search/top-losers-by-chains', this.#baseUrl);
+
+    if (topLosersParams.chains && topLosersParams.chains.length > 0) {
+      url.searchParams.append('chains', topLosersParams.chains.join());
+    }
+    if (topLosersParams.limit) {
+      url.searchParams.append('limit', topLosersParams.limit);
+    }
+
+    const response = await fetch(url, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+
+    if (!response.ok) {
+      throw new Error(
+        `Portfolio API request failed with status: ${response.status}`,
```

Comment:
maybe we could  push forward the error message also here and not just the status? It could help debugging

Reply:
I agree, and I've made a ticket in our backlog to revisit: https://consensyssoftware.atlassian.net/browse/MMPD-1551
Will probably be a fast follow depending on priority.